packages/firestore/test/integration/util/helpers.ts

```diff
@@ -197,6 +207,28 @@ export function withTestDoc(
   });
 }
 
+// TODO(rsgowman): Modify withTestDoc to take in (an optional) initialData and
+// fix existing usages of it. Then delete this function. This makes withTestDoc
+// more analogous to withTestCollection and eliminates the pattern of
+// `withTestDoc(..., docRef => { docRef.set(initialData) ...});` that otherwise is
+// quite common.
```

Comment:
Why is this a TODO rather than just doing it? :-)

Reply:
Because fixing existing uses of it seems big(ish) and otherwise orthogonal to this PR. But since you asked so nicely: https://github.com/firebase/firebase-js-sdk/pull/675